### PR TITLE
Tiler junction layer

### PR DESCRIPTION
**Pull request by rajatSinha303**

### CODE REVIEW

File: TilerServer/TilerWebModule.pas [hunk 1]
```diff
@@ -3248,6 +3274,236 @@ function TSliceLocation.HandleSliceUpdate(const aBuffer: TByteBuffer; var aCurso
...
+
+function TSliceJunctionsPie.getDataValueAtPoint(const aLat, aLon: Double; var aValue: Double): TGenerateTileStatus;
+var
+  isgop: TPair<TWDID, TSliceLocationObject>;
+begin
+  fDataLock.BeginRead;
+  try
+    for isgop in fLocations do
+    begin
+      if isgop.Value.extent.Contains(aLon, aLat) then
```
Comment (cornelissenja) [inline TilerServer/TilerWebModule.pas]: Does extent have width and height or does uit only contain the point. If the latter is true then Contains will never return true and maybe an inline inflate with a small delta would make it work?

File: TilerServer/TilerWebModule.pas [hunk 2]
```diff
@@ -3248,6 +3274,236 @@ function TSliceLocation.HandleSliceUpdate(const aBuffer: TByteBuffer; var aCurso
...
+      begin
+        // test geometry, exit(isgop.Value) if within
```
Comment (cornelissenja) [inline TilerServer/TilerWebModule.pas:3402]: typo: if=is

**Review by cornelissenja — APPROVED** [review]
Small comments added, rest looks good

File: TilerServer/TilerWebModule.pas (hunk 2 above)
Comment (rajatSinha303) [reply TilerServer/TilerWebModule.pas:3402]: done

File: TilerServer/TilerWebModule.pas (hunk 1 above)
Comment (rajatSinha303) [reply TilerServer/TilerWebModule.pas]: fixed

**Pull request merged**